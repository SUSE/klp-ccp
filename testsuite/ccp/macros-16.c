#define G g

static void G(void)
{}

#undef G
#include "macros-16.h"

void pu_f(void)
{
	g();
	h();
}
