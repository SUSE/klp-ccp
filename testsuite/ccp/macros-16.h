#if defined(G)

static void h(int)
{}

#else

static void h(void)
{}

#endif
