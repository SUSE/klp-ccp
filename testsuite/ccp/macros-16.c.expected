/* klp-ccp: from testsuite/ccp/macros-16.c */
#define G g

static void G(void)
{}

#undef G
#include "macros-16.h"

void klpp_pu_f(void)
{
	g();
	h();
}
